lib/core/unittest/CDetachedProcessSpawnerTest.cc

```diff
@@ -64,7 +64,7 @@ BOOST_AUTO_TEST_CASE(testSpawn) {
 
     BOOST_TEST_REQUIRE(spawner.spawn(PROCESS_PATH1, args));
 
-    // Expect the copy to complete in less than 1 second
+    // Expect the copy to complete in less than 1 seconds
```

Comment:
It's probably best to delete the final "s" so that this PR doesn't pointlessly appear in the change history of this file.